simulator/src/main/java/com/github/benmanes/caffeine/cache/simulator/policy/Policy.java

```diff
@@ -48,10 +49,6 @@ default void finished() {}
       return ImmutableSet.of();
     }
 
-    @Override default void record(AccessEvent event) {
```

Comment:
This defeats the purpose of having this adapter. Either the change is wrong or the adapter has to be removed. It was introduced to make it less invasive for existing policies who don't care about the full event.

Reply:
I'll change it back, and it should work fine with the change to PolicyActor

Reply:
changed back in the last commit